fv3core/stencils/d_sw.py

```diff
@@ -359,63 +367,21 @@ def ke_horizontal_vorticity(
         )
 
 
-def initialize_heat_source(heat_source, diss_est):
-    heat_source[grid().compute_interface()] = 0
-    diss_est[grid().compute_interface()] = 0
-
-
-def heat_from_damping(
-    ub,
-    vb,
-    ut,
-    vt,
-    u,
-    v,
-    delp,
-    fx,
-    fy,
-    heat_source,
-    diss_est,
-    kinetic_energy_fraction_to_damp,
-    kstart,
-    nk,
-):
-    heat_source_from_vorticity_damping(
-        ub,
-        vb,
-        ut,
-        vt,
-        u,
-        v,
-        delp,
-        grid().rsin2,
-        grid().cosa_s,
-        grid().rdx,
-        grid().rdy,
-        heat_source,
-        diss_est,
-        kinetic_energy_fraction_to_damp,
-        int(spec.namelist.do_skeb),
-        origin=(grid().is_, grid().js, kstart),
-        domain=(grid().nic, grid().njc, nk),
-    )
-
-
-def set_low_kvals(col):
+def set_low_kvals(col, k):
     for name in ["nord", "nord_w", "d_con"]:
-        col[name] = 0
-    col["damp_w"] = col["d2_divg"]
+        col[name][k] = 0
+    col["damp_w"][k] = col["d2_divg"][k]
 
 
-def vort_damp_option(col):
+def vort_damp_option(col, k):
```

Comment:
Could rename this if it's local. Since it's a lot of python code maybe it's just going to be removed, but it's also easier to remove/refactor if it's clearer.
```suggestion
def vorticity_damping_option(column, k):
```

Reply:
ok sure, I think 'vort', 'damp' and 'col' are fairly obvious but I don't mind expanding them